ruby-challenges.rb

```diff
@@ -19,9 +28,51 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def sumOfNumbers(arr)
+  arr.sum
+end
+
+p sumOfNumbers(nums_array1)
+p sumOfNumbers(nums_array2)
+
+
+# --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0.
+# Create a bike_info method that returns a sentence with all the data from the bike object.
+class Bike
+  attr_accessor :model, :wheels, :current_speed
+  def initialize(model, wheels, current_speed)
+    @model = 'Trek Bike'
```

Comment:
Instead of hard coding model, u could make it dynamic and that way make a new object and have many new instances of the class. ex: @model = model.